setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
The existing package will need to be uninstalled to avoid conflicts with these changes.